packages/forma-36-react-components/src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -1,53 +1,67 @@
-    component: Checkbox,
-  })
-  .add('default', () => (
+export default {
+  title: 'Components/Checkbox',
+  component: Checkbox,
+  parameters: {
+    propTypes: [Checkbox['__docgenInfo']],
+  },
+  argTypes: {
+    className: { control: { disable: true } },
+    testId: { control: { disable: true } },
+    checked: { control: { disable: false } },
+    required: { control: { disable: false } },
+    disabled: { control: { disable: false } },
+  },
+};
+
+export const basic = (args: CheckboxProps) => (
+  <Checkbox id="Checkbox" labelText={args['labelText']} name={args.name} />
+);
+
+export const overview = () => (
+  <>
+    <Flex marginBottom="spacingS" marginTop="spacingM">
+      <SectionHeading element="h3">Checkbox default</SectionHeading>
+    </Flex>
+    <Checkbox id="Checkbox" labelText="some label text" name="some-name" />
+    <Flex marginBottom="spacingS" marginTop="spacingM">
+      <SectionHeading element="h3">Checkbox checked</SectionHeading>
+    </Flex>
+    <Checkbox
+      id="Checkbox"
+      checked
+      labelText="some label text"
+      name="some-name"
+    />
+    <Flex marginBottom="spacingS" marginTop="spacingM">
+      <SectionHeading element="h3">Checkbox disabled</SectionHeading>
+    </Flex>
     <Checkbox
-      className={text('className', '')}
       id="Checkbox"
-      checked={boolean('checked', false)}
-      labelText={(text('labelText'), 'some label text')}
-      disabled={boolean('disabled', false)}
-      required={boolean('required', false)}
-      name={text('name', 'some-name')}
-      onChange={action('onChange')}
-      willBlurOnEsc={boolean('willBlurOnEsc', true)}
+      disabled
+      labelText="some label text"
+      name="some-name"
     />
-  ))
-  .add('overview', () => (
-    <>
-      <Flex marginBottom="spacingS" marginTop="spacingM">
-        <SectionHeading element="h3">Checkbox default</SectionHeading>
-      </Flex>
-      <Checkbox id="Checkbox" labelText="some label text" name="some-name" />
-      <Flex marginBottom="spacingS" marginTop="spacingM">
-        <SectionHeading element="h3">Checkbox checked</SectionHeading>
-      </Flex>
-      <Checkbox
-        id="Checkbox"
-        checked
-        labelText="some label text"
-        name="some-name"
-      />
-      <Flex marginBottom="spacingS" marginTop="spacingM">
-        <SectionHeading element="h3">Checkbox disabled</SectionHeading>
-      </Flex>
-      <Checkbox
-        id="Checkbox"
-        disabled
-        labelText="some label text"
-        name="some-name"
-      />
-    </>
-  ));
+    <Flex marginBottom="spacingS" marginTop="spacingM">
+      <SectionHeading element="h3">Checkbox disabled checked</SectionHeading>
+    </Flex>
+    <Checkbox
+      id="Checkbox"
+      disabled
+      checked
+      labelText="some label text"
+      name="some-name"
+    />
+  </>
+);
+
+basic.args = {
+  id: 'Checkbox',
+  labelText: 'some label text',
+  name: 'some name',
+};
```

Comment:
I would keep the default values close to the story, it's easier to know how the story will look without scrolling to the bottom of the file